lib/activerecord-multi-tenant/model_extensions.rb

```diff
@@ -23,13 +23,21 @@ def partition_key
           # Avoid primary_key errors when using composite primary keys (e.g. id, tenant_id)
           def primary_key
             return @primary_key if @primary_key
-            return @primary_key = super || DEFAULT_ID_FIELD if ::ActiveRecord::VERSION::MAJOR < 5
+
+            if ::ActiveRecord::VERSION::MAJOR < 5
+              @primary_key = super || DEFAULT_ID_FIELD
+              return @primary_key if connection.schema_cache.columns_hash(table_name).include? @primary_key else nil
```

Comment:
I think this can be written like this for more idiomatic Ruby:

```ruby
return @primary_key if connection.schema_cache.columns_hash(table_name).include?(@primary_key)
```
(no need for the else since this is returning from the function)

Reply:
thanks :)